### PR TITLE
ARC-690 noop virus scanning

### PR DESCRIPTION
**Pull request by atahmasebiupgrade**
This PR disable virus scanning while we figure out UI's behavior to handle the delays introduced by virus scanning. 

When we need to enable it, we could just set the env variable `IS_AV_ENABLED` to true.

### CODE REVIEW

**Comment by gjacquetupgrade** [comment]
@jpbelangerupgrade the idea was more to have a feature flag to disable scanning in case it starts causing issues.

**Comment by jpbelangerupgrade** [comment]
hmmmmm still on the fence for that. Guess there's a use case for this. just see this as a high risk change. Letting un-scanned file thru our pipeline.

**Comment by jpbelangerupgrade** [comment]
ahhh context is everything :) 

if this goes to prod.. and we want it disabled. then the flag is fine. Since this PR had no description of ticket/context was just trying to make sure this wasn't a temporary flag that could cause problem in prod.

**Comment by atahmasebiupgrade** [comment]
@jpbelangerupgrade @gjacquetupgrade So, the question is do we have to merge this? or maybe we can just deploy the lambda from this branch?

**Comment by gjacquetupgrade** [comment]
for dev, you can deploy from your laptop for now, for other environments, I would much rather deploy from a tag.

File: common.py [hunk 1]
```diff
@@ -30,6 +30,7 @@
 CLAMSCAN_PATH = os.getenv("CLAMSCAN_PATH", "./bin/clamscan")
 FRESHCLAM_PATH = os.getenv("FRESHCLAM_PATH", "./bin/freshclam")
 AV_PROCESS_ORIGINAL_VERSION_ONLY = os.getenv("AV_PROCESS_ORIGINAL_VERSION_ONLY", "False")
+IS_AV_ENABLED = os.getenv("IS_AV_ENABLED", "False")
```
Comment (gjacquetupgrade) [inline common.py]: Can you default it to true?

**Pull request merged**